build.gradle.kts

```diff
@@ -45,6 +45,7 @@ intellijPlatform {
     version = provider { project.version.toString() }
     ideaVersion {
       sinceBuild = "242"
+      untilBuild = "242.*"
     }
```

Comment:
https://github.com/aSemy/ij-problem-ignorer/blob/v0.0.4/build.gradle.kts#L44-L47

This is the same config I had in v0.0.4 though :(

Reply:
I’ll be watching over the weekend, can you tell me how I can download the plugin locally?

Reply:
This repo uses the official IJ Gradle Plugin, so all the info should be in the docs :)

https://plugins.jetbrains.com/docs/intellij/tools-intellij-platform-gradle-plugin.html

Reply:
> This repo uses the official IJ Gradle Plugin, so all the info should be in the docs :)
> 
> https://plugins.jetbrains.com/docs/intellij/tools-intellij-platform-gradle-plugin.html

FIxed

Reply:
<img width="800" alt="image" src="https://github.com/user-attachments/assets/f84812a6-374f-484b-a44c-d5516b8007b5">